script.js

```diff
@@ -303,3 +311,36 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
     "</div>"
   );
 }
+
+function initializeDragAndDrop () {
+  $(".js-board").sortable({
+    connectWith: ".js-board",
+    stop: function(event, ui) {
+      updatePivotal(ui);
+    }
+  });
+} //end initializeDragAndDrop
+function updatePivotal(ui) {
+    let id = ui.item.data("id");
+    console.log(ui.item);
+    let newStatus = ui.item.closest(".js-board").data("status");
+    let beforeId = ui.item.prev().data("id");
+    let urlId = "/" + ui.item.data("id");
+    if (beforeId === undefined) beforeId = null;
+    instance.put("/" + id, {
+    "current_state": newStatus,
+    "before_id": beforeId
```

Comment:
Not sure where my comment went, but what happens if you move the card to the first position in the list and there's nothing before it?

Reply:
i dont get any errors and they look like they are in order on my app